### PR TITLE
feat: implement SfRange component

### LINKED ISSUE #852: [FEATURE] Implement SfRange
According to the new design, we need to implement new component SfScale

image

Design: https://www.figma.com/file/N0Ct95cSAoODNv7zYS01ng/Storefront-UI-Design-System?node-id=187%3A336

### PR DESCRIPTION
**Pull request by AdamPawlinski**
# Related issue
Closes #852

# Scope of work
Implement new component SfRange.
It contnue work fromt his PR https://github.com/vuestorefront/storefront-ui/pull/1230 

# Screenshots of visual changes

# Checklist

- [x] No commented blocks of code left
- [x] Run tests and docs
- [x] Self code-reviewed
- [x] Changes documented 

If applicable:

- [ ] I followed [composition rules](https://docs.storefrontui.io/?path=/story/introduction-contributing-guide-code-guidelines--page) for my component
- [ ] I tested the component in most common device sizes (can be tested in Storybook [from viewport addon in top menu](https://github.com/storybooks/storybook/tree/master/addons/viewport))

### CODE REVIEW

File: packages/vue/src/components/molecules/SfRange/SfRange.stories.js [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+    },
+    orientation: {
+      control: "text",
+      table: {
+        category: "Props",
+      },
+      defaultValue: "horizontal",
+    },
+    behaviour: {
+      control: "text",
```
Comment (justyna-13) [inline packages/vue/src/components/molecules/SfRange/SfRange.stories.js:70]: also here would be good to know what options are available :) maybe we can add descriptions for props with more options to choose :) or link to docs?

File: packages/vue/src/components/molecules/SfRange/SfRange.stories.js [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+  data() {
+    return {
+      rangeValue: this.value,
+    };
+  },
+  props: Object.keys(argTypes),
+  template: `
+  <div style="padding: 100px">
+    <SfRange
+      v-model="rangeValue"
```
Comment (justyna-13) [inline packages/vue/src/components/molecules/SfRange/SfRange.stories.js:126]: there is a wrong prop type in a console :)

**Review by justyna-13 — COMMENTED** [review]
Great job! It looks great :) see my comments

File: packages/vue/src/components/molecules/SfRange/SfRange.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+# component-description
```
[the remark this answers is not in the document]
Comment (AdamPawlinski) [reply packages/vue/src/components/molecules/SfRange/SfRange.md:1]: No, we don't need it, you are right :)

File: packages/vue/src/components/molecules/SfRange/SfRange.stories.js [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+    },
+    connect: {
+      control: "boolean",
+      table: {
+        category: "Props",
+      },
+      defaultValue: true,
+    },
+    direction: {
+      control: "text",
```
[the remark this answers is not in the document]
Comment (AdamPawlinski) [reply packages/vue/src/components/molecules/SfRange/SfRange.stories.js:56]: Of course you are right. I change that.

File: packages/vue/src/components/molecules/SfRange/SfRange.stories.js [hunk 5]
```diff
@@ -0,0 +1,181 @@
...
+      table: {
+        category: "Props",
+      },
+      defaultValue: "ltr",
+    },
+    orientation: {
+      control: "text",
```
[the remark this answers is not in the document]
Comment (AdamPawlinski) [reply packages/vue/src/components/molecules/SfRange/SfRange.stories.js:63]: I changed it to radio.

File: packages/vue/src/components/molecules/SfRange/SfRange.stories.js (hunk 2 above)
Comment (AdamPawlinski) [reply packages/vue/src/components/molecules/SfRange/SfRange.stories.js:126]: Thanks :)

**Comment by AdamPawlinski** [comment]
I made all changes, thanks @justyna-13

**Comment by Baroshem** [comment]
@AdamPawlinski resolve conflicts and fix tests please :)

**Review by Baroshem — APPROVED** [review]
Awesome work!

**Review by justyna-13 — APPROVED** [review]
great! :rocket:

**Pull request merged**